docs/blog/2019-04-03-how-we-think-about-product-at-gatsby/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "How we think about product at Gatsby"
+date: 2019-04-03
+author: Marisa Morby
+excerpt: "How do we make decisions about products at Gatsby? Take a look behind the scenes to see how we're using outcome-based planning, OKRs, and other strategies to create the best possible products for our community."
+tags:
+  - company
+  - commercial
+  - product
+---
+
+As a company grows, it will eventually start offering products or services to the community it’s trying to serve. It’s an exciting time filled with near constant changes. At Gatsby, we realized that this meant we needed to have a better understanding and core philosophy around the products we create.
+
+So, we blocked out our schedules and met for 3 days over [Zoom](https://zoom.us/), with people calling in from different parts of the USA, Germany, and England to read, workshop, and talk about product.
+
+On the call, we read excerpts from Marty Cagan’s [_Inspired: How to Create Tech Products Customers Love_](https://svpg.com/inspired-how-to-create-products-customers-love/). We’re using a lot of his advice to help create a product organization and product management framework at Gatsby.
+
+## What is a product and what are Gatsby products?
+
+After a lot of back and forth, talking through examples, and working through questions, we landed on what we think of as “a product” at Gatsby.
+
+A product is an individual thing (software, hardware, etc.) that facilitates stable customer behaviors and is scalable.
+
+Additionally, a product must add value in order for a customer to continually use it. At Gatsby, we believe that we add value by solving real and impactful customer problems for individuals and teams in the web development community.
+
+## What is our product organization?
+
+A product organization is the group of people responsible for thinking about what we offer our customers and why. Our product organization, or product org for short, currently consists of a group of product managers, designers, and UX researchers.
```

Comment:
It seems like we could mention people by name here!

Reply:
I didn't mention people by name since we're going to be growing, and I wanted to make this as evergreen as possible.